src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -139,7 +139,8 @@ export class BaseEditCourseForm extends React.Component {
       return (
         <>
           <Hyperlink
-            destination={`https://www.edx.org/preview/course/${courseInfo.data.url_slug}`}
+            name="preview-url"
+            destination={`https://www.edx.org/preview/${courseInfo?.data?.url_slug.includes('/') ? '' : 'course/'}${courseInfo.data.url_slug}`}
```

Comment:
Please remove hardcoded edx.org/preview and move this to a setting/env var. Never realized this was hardcoded.

Reply:
fixed.
`edx-internal` PR: https://github.com/edx/edx-internal/pull/8625